<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagingWidget</class>
 <widget class="QWidget" name="PagingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>41</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <spacer name="horizontalSpacer_Left">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>60</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_Left">
     <property name="text">
      <string>|&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Previous">
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="Line_CurrentPage">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_MaxPage">
     <property name="text">
      <string>/100</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Next">
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_Right">
     <property name="text">
      <string>&gt;|</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_Right">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>59</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
